addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/sagemaker-notebook-instance.cfn.yml

```diff
@@ -94,6 +97,25 @@ Resources:
                 - logs:CreateLogGroup
               Resource:
                 - !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/sagemaker/*
+
+  # This policy is attached to the role after the instance is created
```

Comment:
Nice call! Can we rename 'IAMRole' to something more specific?

Reply:
The RoleName is assigned according to the namespace, which makes it deployment-specific appended by the workspace type. (See line 51)

Reply:
Yes! I was talking about the name of the resource in the template itself? '- !Ref IAMRole' seems generic and I spent sometime trying to figure out how this policy gets attached to the SageMaker role after the instance is created. If it says something like '- !Ref SageMakerIAMRole', it would be self-explanatory and easier to grasp. Just some random thoughts, it's very minor.